android/modules/ui/res/drawable/fastscroll_line.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
+       android:shape="rectangle">
+
+    <solid android:color="@android:color/darker_gray" />
+
+    <padding
+        android:top="10dp"
+        android:left="10dp"
+        android:right="10dp"
+        android:bottom="10dp"/>
+</shape>
```

Comment:
The "res" xml files you are adding should be prefixed with either "titanium_" or "ti_" to avoid name collision.